pages/base.page.js

```diff
@@ -0,0 +1,38 @@
+const PAGE_TIMEOUT = 10000
+
+export default class BasePage {
+
+
+   /**
+    * Abrir página
+    * @param {String} ruta a la cual acceder
+    */
+   async abrir(ruta) {
```

Comment:
Remember we defined the convention of **everything in english**, this includes method names.

Reply:
The convention of everything in english was defined after I started the project, but I will correct the names.